src/controllers/user.controller.ts

```diff
@@ -72,4 +75,18 @@ export async function getUser(req: Request, res: Response) {
   } else {
     res.sendStatus(404);
   }
+}
+
+export async function editUser(req: Request, res: Response) {
+  const id = req.params.id;
```

Comment:
what's the edit user method for? Ohh I see, it updates the user, perhaps a different name for the function would be more suitable?